submissions/ArthurGorbenko/Memory Pair Game/js/script.js

```diff
@@ -0,0 +1,128 @@
+const valuesOfBackPos = [
```

Comment:
for `const` write `VALUES_OF_BACK_POS`
You might ask why I need this if I write `const`
Because inside the code, I do not know if this variable is a constant or a simple variable))
This applies to all constants in your code

Reply:
Renamed.